src/blocks/services/styles/editor.scss

```diff
@@ -0,0 +1,59 @@
+.wp-block-coblocks-services {
+	[data-block] {
+		margin-bottom: 0;
+		margin-top: 0;
+	}
+
+	.block-editor-inner-blocks {
+		width: 100%;
+	}
+
+	.block-editor-block-list__layout {
+		display: flex;
+		flex-flow: row nowrap;
+		justify-content: space-between;
+
+		// On small screens, we are no longer using row direction but column.
+		@media all and (max-width: 514px) {
+			flex-direction: column;
+		}
+
+		.block-editor-block-list__layout {
+			display: block;
+		}
+	}
+
+	.wp-block[data-type="coblocks/service"] {
+		flex: 1 100%;
+		max-width: 100%;
+
+		> .block-editor-block-list__block-edit {
+			margin-top: 1em;
```

Comment:
What are these margin and bottom parts applied for? They seem to be affecting the block selection border.

Reply:
This was put in place to make the block look the same within the editor and the theme. I'm not satisfied with this either so I'm going to find a different solution.